packages/@webex/plugin-cc/src/services/core/types.ts

```
@@ -0,0 +1,48 @@
+      };
+  data?: RequestBody;
+  headers?: Record<string, string>;
+  timeout?: Timeout;
+  notifCancel?: {bind: Bind; msg: TRes};
+};
+
+export type Conf<TRes, TErr, TReq> = (p: TReq) => Req<TRes, TErr>;
+export type ConfEmpty<TRes, TErr> = () => Req<TRes, TErr>;
+export type Res<TRes, TReq> = (p: TReq, cbRes?: CbRes<TRes>) => Promise<TRes>;
+export type ResEmpty<TRes> = (cbRes?: CbRes<TRes>) => Promise<TRes>;
+export type CbRes<TRes> = (res: any) => void | TRes;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)